command/scaling_policy_info.go

```diff
@@ -0,0 +1,126 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+// Ensure ScalingPolicyInfoCommand satisfies the cli.Command interface.
+var _ cli.Command = &ScalingPolicyInfoCommand{}
+
+// ScalingPolicyListCommand implements cli.Command.
+type ScalingPolicyInfoCommand struct {
+	Meta
+}
+
+// Help satisfies the cli.Command Help function.
+func (s *ScalingPolicyInfoCommand) Help() string {
+	helpText := `
+Usage: nomad scaling policy info [options] <policy_id>
+
+  Info is used to read the specified scaling policy.
+
+General Options:
+
+  ` + generalOptionsUsage() + `
+
+Policy Info Options:
+
+  -json
+    Output the scaling policy in its JSON format.
+
+  -t
+    Format and display the scaling policy using a Go template.
+`
+	return strings.TrimSpace(helpText)
+}
+
+// Synopsis satisfies the cli.Command Synopsis function.
+func (s *ScalingPolicyInfoCommand) Synopsis() string {
+	return "Display an individual Nomad scaling policy"
+}
+
+func (s *ScalingPolicyInfoCommand) AutocompleteFlags() complete.Flags {
+	return mergeAutocompleteFlags(s.Meta.AutocompleteFlags(FlagSetClient),
+		complete.Flags{
+			"-json": complete.PredictNothing,
+			"-t":    complete.PredictAnything,
+		})
+}
+
+// Name returns the name of this command.
+func (s *ScalingPolicyInfoCommand) Name() string { return "scaling policy info" }
+
+// Run satisfies the cli.Command Run function.
+func (s *ScalingPolicyInfoCommand) Run(args []string) int {
+	var json bool
+	var tmpl string
+
+	flags := s.Meta.FlagSet(s.Name(), FlagSetClient)
+	flags.Usage = func() { s.Ui.Output(s.Help()) }
+	flags.BoolVar(&json, "json", false, "")
+	flags.StringVar(&tmpl, "t", "", "")
+	if err := flags.Parse(args); err != nil {
+		return 1
+	}
+
+	if args = flags.Args(); len(args) != 1 {
+		s.Ui.Error("This command takes one argument: <policy_id>")
+		s.Ui.Error(commandErrorText(s))
+		return 1
+	}
+
+	// Get the policy ID.
+	policyID := args[0]
+
+	// Get the HTTP client.
+	client, err := s.Meta.Client()
+	if err != nil {
+		s.Ui.Error(fmt.Sprintf("Error initializing client: %s", err))
+		return 1
+	}
+
+	policy, _, err := client.Scaling().GetPolicy(policyID, nil)
```

Comment:
We should probably add something like `GetPolicyByPrefixID` so this command can be used with partial ID prefix matching.

Reply:
Agreed.